{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "d8cb5a12_1a5e778a",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1120458
      },
      "writtenOn": "2021-04-09T20:17:11Z",
      "side": 1,
      "message": "Several of the tests failing here, I can confirm they pass locally. I believe this was racing w/ LKGB and another fix.",
      "revId": "1a3a8effca802995ece193b7d40cd80abba55798",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "007a0b14_157906ee",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1120458
      },
      "writtenOn": "2021-04-09T22:41:37Z",
      "side": 1,
      "message": "The failure here is b/184978217. I will wait until this is auto-retried or until Monday, given the nature of this CL.",
      "revId": "1a3a8effca802995ece193b7d40cd80abba55798",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "d6d65a03_6c547635",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 5
      },
      "lineNbr": 0,
      "author": {
        "id": 1120458
      },
      "writtenOn": "2021-04-12T18:07:23Z",
      "side": 1,
      "message": "Presubmit failure is known issue b/185007114.",
      "revId": "1a3a8effca802995ece193b7d40cd80abba55798",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}